controllers/attractor/lb-fe.go

```diff
@@ -104,7 +104,8 @@ func (l *LoadBalancer) getNscEnvVars(allEnv []corev1.EnvVar) []corev1.EnvVar {
 			e.Name == "NSM_NAME" ||
 			e.Name == "NSM_DIAL_TIMEOUT" ||
 			e.Name == "NSM_REQUEST_TIMEOUT" ||
-			e.Name == "NSM_MAX_TOKEN_LIFETIME" {
+			e.Name == "NSM_MAX_TOKEN_LIFETIME" ||
+			e.Name == "NSM_LIVENESSCHECKENABLED" {
```

Comment:
maybe set it to false in go code line 98, so users won't be able to set the value in the template.

Reply:
I thought about it, but probably you are right, that currently changing this value could break the system, so it's better to hardcode it